docs/roles/ako-tenant.json

```diff
@@ -0,0 +1,250 @@
+{
```

Comment:
Why is this ako-tenant.json, shouldn't it be amko-tenant ?

Reply:
it should be amko-tenant. Modified accordingly.